rows.go

```diff
@@ -947,5 +947,5 @@ func convertRowHeightToPixels(height float64) float64 {
 	if height == 0 {
 		return 0
 	}
-	return math.Ceil(4.0 / 3.4 * height)
+	return height * 4.0 / 3.0
```

Comment:
The `math` import of this file also no longer required. Please run test locally by `go test .` after changes.